lua/geyser/GeyserLabel.lua

```diff
@@ -111,9 +111,15 @@ function Geyser.Label:getWindow(label)
         if v.name == label then
             return v
         end
-    end
-end
 
+        -- 030517 the below searchs down one level
```

Comment:
Ohh. No need for a timestamp @Nyyrazzilyss - Git already keeps track of that for you automatically ([see example](https://github.com/vadi2/mudlet-lua/blame/master/lua/DB.lua)). Mind updating the comment to say _why_ it does what it does? That is, enables the master flyout label to be nestable.